devel/addon/certmanager/install.sh

```diff
@@ -25,10 +25,23 @@ RELEASE_NAME="${RELEASE_NAME:-cert-manager}"
 # Default feature gates to enable
 FEATURE_GATES="${FEATURE_GATES:-ExperimentalCertificateSigningRequestControllers=true,ExperimentalGatewayAPISupport=true,AdditionalCertificateOutputFormats=true}"
 
+# As Feature Gates are added/removed, these lists should be updated.
```

Comment:
Do we have documentation on using, adding and removing feature gates? If not, can we add some documentation for contributors including the instruction in this comment?

Reply:
I'm not aware of any focused documentation we have on feature gates, but we should definitely have it. I'll add an issue on the cert-manager website to track this.

Reply:
https://github.com/cert-manager/website/issues/844

Reply:
Thanks!